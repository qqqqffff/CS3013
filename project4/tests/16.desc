eval RR analysis with one job with time slice 4
