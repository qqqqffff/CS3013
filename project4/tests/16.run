./scheduler RR tests/16.in 4
