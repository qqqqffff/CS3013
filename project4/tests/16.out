Begin analyzing RR:
Job 0 -- Response time: 0  Turnaround: 6  Wait: 0
Average -- Response: 0.00  Turnaround: 6.00  Wait: 0.00
End analyzing RR.
